terraform/network.tf

```diff
@@ -0,0 +1,11 @@
+resource "docker_network" "armonik-net" {
```

Comment:
Why 3 networks?

Reply:
That's what I did in the docker-compose files

Reply:
I'm using only one in the last commited version